Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs

```diff
@@ -537,13 +537,22 @@ protected virtual void OnSetDataState(Model.State state)
         // Updates UI based on values on View.Data
         protected virtual void UpdateUI()
         {
+            Debug.Log("TextElipsis: " + TextElipsis(Data.Title));
```

Comment:
please please please check for logs and remove before committing :)

Reply:
or actually, what I do sometimes is go ahead and commit but then read through my PR and look for things like this before inviting others to check the PR

Reply:
I'm so sorry! I thought I had deleted all the debugs :s